solutions/devsprint-attekita-2/RealEstateApp/Screens/PropertyList/PropertyListViewController.swift

```diff
@@ -7,22 +7,42 @@
 
 import UIKit
 
-class PropertyListViewController: UIViewController {
-
+final class PropertyListViewController: UIViewController {
+    // MARK: - Lifecycle Methods
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        navigationController?.navigationBar.prefersLargeTitles = true
-        navigationItem.title = "Real Estate App 🏡"
-
-        self.view.backgroundColor = .white
-
+        
+        setup()
+    }
+    
+    override func loadView() {
+        let emptyView = EmptyView()
```

Comment:
Poderíamos ter um método set emptyView, e chamar ele quando o array do fetchServices for vazio.
Nele nós poderíamos setar a backgroundView da tableview  = EmptyView()
dessa forma será atribuída a view no background da tableview e como ela não possui resultados, teríamos o comportamento especificado no layout.

Outra forma de fazer, seria adicionar a view por cima da tableview setando as constraints, e gerenciar a visibilidade da view de acordo com o resultado do fetchServices